docker-compose.yml

```diff
@@ -3,7 +3,7 @@ version: '3.4'
 services:
   extractor:
     dns: 8.8.8.8
-    image: docker.edu-sharing.com/projects/oeh-redaktion/edusharing-projects-oeh-redaktion-metalookup-extractor:latest
+    image: community.docker.edu-sharing.com/oeh-search-meta:latest
```

Comment:
Please rewind :)

Reply:
done, the lighthouse container was also on `docker.edu-sharing.com ...`